CONTRIBUTORS.md

```diff
@@ -138,6 +138,13 @@ Listed in alphabetical order.
     </td>
     <td>abdadeel_</td>
   </tr>
+  <tr>
+    <td>Abe Hanoka</td>
+    <td>
+      <a href="https://github.com/abe-101">abe-101</a>
+    </td>
+    <td>abe__101</td>
+  </tr>
```

Comment:
it's not necessary to manually update the list of contributors. We have automation via GitHub Actions that does this automatically (once a day when there are new contributors)

Reply:
Good to know
I've removed the commit